### PR TITLE
Update db-auth

### PR DESCRIPTION
**Pull request by Tony931023**
**Update**

* Language change
* Table structures
* Code standards

### CODE REVIEW

**Comment by Tony931023** [comment]
#601

File: docs/account.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [<-Back-to:Auth](database-auth.md)
 
+Read in: [English :gb:](account) [Spanish :es:](es/account)
```
Comment (Kitzunu) [inline docs/account.md:5]: No. I dont think this is a good change. As far as I understand it, if you change on the home page all links would be in that locale. So I do not like this change. Please revert it

Comment (Tony931023) [reply docs/account.md:5]: I don't know, it depends perhaps to allow the user to change the language of the pages more easily for example:
You are on the English wiki inside the acount table in the auth database https://www.azerothcore.org/wiki/account and you want to read that same table in Spanish, it is easier for the user to give it directly there is the same to Spanish than do not change the language in the sidebar look for db-auth and there is inside the file that contains the tables look for the acoount table to read it in Spanish.

If you think that example is not correct, tell me and I will remove the language change

Comment (Kitzunu) [reply docs/account.md:5]: I dont like it. Maybe it is possible to make the sidebar language thing to also force the current webpage to go to en/es

Comment (Tony931023) [reply docs/account.md]: all ready @Kitzunu https://github.com/azerothcore/wiki/pull/602/commits/305e067fa90ea1ae398d5b6740617e2acfb851b9

File: docs/account.md [hunk 2]
```diff
@@ -108,45 +108,45 @@ Make sure the PHP GMP extension is loaded! Uncomment `extension=gmp` in your php
 
 [VerifySRP6Login.php](https://gist.github.com/Treeston/34d9249fb467dddc11b2568e74f8cb1e)
 
-### session_key
+### session\_key
```
Comment (FrancescoBorzi) [inline docs/account.md:111]: why is this better? just wondering

Comment (Kitzunu) [reply docs/account.md:111]: It doesnt matter. Before with first markdown we used we had to do it to escape underline.

Now with the one we use this isnt needed, but doesnt matter if it is added or not

Comment (Tony931023) [reply docs/account.md:111]: This is the pull list, now you can give it merge

File: docs/database-auth.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Database Auth
 
+Read in: [English :gb:](database-auth) [Spanish :es:](es/database-auth)
```
Comment (Kitzunu) [inline docs/database-auth.md:3]: no

Comment (Tony931023) [reply docs/database-auth.md]: Fix https://github.com/azerothcore/wiki/pull/602/commits/2adf44bbdbbc74bf82e5d4fefa489f736b898dfa

**Comment by FrancescoBorzi** [comment]
@Tony931023  @Kitzunu what's the status of this?

**Comment by Tony931023** [comment]
@FrancescoBorzi already 100%

**Comment by Kitzunu** [comment]
I will review on Sunday, only free day this week. Sorry

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**